Effisense/sensor-dashboard
# Feat/#4 organization owner management

Closes: #4

## Checklist


## Changelog

- Made a branch from issue #3  
-  User visits /, and gets redirected to sign-in.
- If user is already logged in, user gets redirected to dashboard.
-  Use sign in and sign up component from Clerk
- User can log out. Use Clerk component already present in index page.
- Design is not finished with the sign in/ login, waiting on prototype
- Admin sets an existing user to an organization owner. An organization owner can use the OrganizationSwitcher to toggle which organization they're currently in.
- The organization owner gets a button in the navbar for "Manage organization", which redirect to organizations/[id] and renders the OrganizationProfile from Clerk.

## How to test
- Make sure that your user is already in a organization. Visit Clerk to do this: https://dashboard.clerk.dev/apps/app_2MdLtYV4QI3RffKSTpxkebX1F4H/instances/ins_2MdLtioKh2ePIn6TYnzi6RxOagV/organizations
- Make sure that you get redirected to the sign in page if you are not logged in. 
- Make sure that you get redirected to the dashboard if you are logged in. 
- Observe that the UserButton in the navbar works, you should be able to click your avatar to see a user profile. 
- Observe that you can use the OrganizationSwitcher in the navbar to toggle between which organization you are logged into.
- Make sure that if you are admin of that organization that you are able to see the "manage organization" button inside the OrganizationSwitcher, and that it renders to the OrganizationProfile where you can manage the organization.
